bitnami/minio/README.md

```diff
@@ -87,7 +87,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `mode`                     | MinIO&reg; server mode (`standalone` or `distributed`)                                                                                                                                                           | `standalone`             |
 | `auth.rootUser`            | MinIO&reg; root username                                                                                                                                                                                         | `admin`                  |
 | `auth.rootPassword`        | Password for MinIO&reg; root user                                                                                                                                                                                | `""`                     |
-| `auth.existingSecret`      | Use existing secret for credentials details (`auth.rootUser` and `auth.rootPassword` will be ignored and picked up from this secret). The secret has to contain the keys `root-user` and `mariadb-root-password) | `""`                     |
+| `auth.existingSecret`      | Use existing secret for credentials details (`auth.rootUser` and `auth.rootPassword` will be ignored and picked up from this secret). The secret has to contain the keys `root-user` and `root-password`) | `""`                     |
```

Comment:
The README table is autogenerated based on the metadata available in the **values.yaml** using the [eadme-generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool. Therefore, you need to fix this in the values.yaml (check the line below) and run the tool to update the table:

- https://github.com/bitnami/charts/blob/master/bitnami/minio/values.yaml#L97

Reply:
Alright, thanks @juan131 . Could you point to the tool to generate the readme? I couldn't find any...

Reply:
I added a link in my comment, here you have it in any case: https://github.com/bitnami-labs/readme-generator-for-helm